content/sensu-go/5.20/reference/backend.md

```diff
@@ -417,6 +436,23 @@ sensu-backend start --deregistration-handler /path/to/handler.sh
 deregistration-handler: "/path/to/handler.sh"{{< /highlight >}}
 
 
+| labels     |      |
+-------------|------
+description  | Custom attributes to include with event data that you can use for response and dashboard view filtering.<br><br>If you include labels in your event data, you can filter [API responses][25], [sensuctl responses][26], and [dashboard views][54] based on them. In other words, labels allow you to create meaningful groupings for your data.<br><br>Limit labels to metadata you need to use for response filtering. For complex, non-identifying metadata that you will *not* need to use in response filtering, use annotations rather than labels.
+required     | false
+type         | Map of key-value pairs. Keys can contain only letters, numbers, and underscores and must start with a letter. Values can be any valid UTF-8 string.
+default      | `null`
+environment variable | `SENSU_LABELS`
+example               | {{< highlight shell >}}# Command line examples
+sensu-backend start --labels proxy_type=website
```

Comment:
This `proxy_type` example makes sense as an agent label, but not really as a backend label. I would suggest a different example such as `security_zone=us-west-2a` or something more applicable to a real-world use case of a backend entity label.